data/watermarks/simple-text.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     x="0"
+     y="0">
+
+  <text x="0" y="5px" text-anchor="start"
+        fill="#$(WATERMARK_COLOR)"
+        font-family="$(WATERMARK_FONT_FAMILY)"
+        font-style="$(WATERMARK_FONT_STYLE)"
+        font-weight="$(WATERMARK_FONT_WEIGHT)"
+        font-size="10px">$(WATERMARK_TEXT)</text>
```

Comment:
I was mostly asking because later on you:
`g_strlcpy(n->font, "DejaVu Sans 10", sizeof(n->font));`
10 is not the font size there?